src/components/molecules/NavSearchBar/NavSearchBar.tsx

```diff
@@ -59,9 +63,12 @@ const NavSearchBar = () => {
       event.name.toLowerCase().includes(normalizedSearchQuery)
     );
 
-    const roomsResults: Room[] = (venue?.rooms?.filter((room) =>
-      room.title.toLowerCase().includes(normalizedSearchQuery)
-    ) ?? []) as Room[]; // @debt Clean up this hack properly once old templates are deleted
+    const roomsResults: Room[] =
+      venue?.rooms
+        ?.filter((room) =>
+          room.title.toLowerCase().includes(normalizedSearchQuery)
+        )
+        .filter((room) => room.type !== RoomTypes.unclickable) ?? [];
```

Comment:
We can handle both of these `.filter` cases in a single pass, rather than iterating through the collection twice.

We could also likely remove this at the query level, rather than fetching them all then filtering on the frontend.

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1225/commits/03ee9c9b63c7abd63465d57951e08c49b50653d9